### DISCUSSION TITLE
print argument

### DISCUSSION THREAD

**Discussion by bellaguin** [charnley/rmsd#120] [Q&A]
Hello, thank you for making this tool! When I run `calculate_rmsd FILE_A FILE_B --print`, the resulting output contains XYZ coordinates, but with atomic numbers in the place of element symbols. Is there a reason for doing this, as it seems non-standard for XYZ files?

**Comment by rafaelpap** [comment]
Line 2056 of calculate_rmsd.py is:
            get_coordinates_xyz, is_gzip=settings.format_is_gzip, return_atoms_as_int=True
if you change True to False the atomic symbol is printed.
I cannot comment on the rationale of being True by default.

**Reply by charnley** [reply]
I agree, for printing, it should default to symbols. The reason for using ``int`` internally, is because it is easier/faster to navigate numpy arrays with integers.

**Comment by charnley** [comment]
@bellaguin , you feedback is correct. Using integers is non-standard (but allowed). Newest version is change to use symbols on ``--print``.
